pkgs/firehose/test/changelog_test.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:firehose/src/changelog.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('changelog', () {
+    late Changelog changelog;
+
+    setUp(() {
+      changelog = Changelog(File('CHANGELOG.md'));
```

Comment:
Maybe generate the file here instead of reading the actual changelog? This way, you could also test for the absence of version or latest changes...

Reply:
+1, will do